krux_boto/boto.py

```diff
@@ -192,6 +190,26 @@ def __init__(
         if region is None:
             region = DEFAULT['region']()
 
+        # GOTCHA: Due to backward incompatible version change, the users of krux_boto may pass wrong credential
```

Comment:
lets' refer to the versions that this change happened - over time this comment will become misleading otherwise.